verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -67,15 +67,18 @@ interface TableRow {
     admittedToHospital: string;
 }
 
-interface Props {
+interface LocationState {
+    newCaseIds: [string];
```

Comment:
Intentional `[tuple]` in lieu of `array[]`?

Reply:
Nope, thanks, changed.